frontend/viewdir/template.njk

```diff
@@ -6,22 +6,14 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
```

Comment:
We might want to anchor the url to the root with a preceding `/`. At the least it makes it consistent with the urls to the images

```suggestion
    <link rel="stylesheet" type="text/css" href="/css/style.css" />
```